@here/harp-mapview/lib/MapView.ts

```diff
@@ -1921,8 +1934,16 @@ export class MapView extends THREE.EventDispatcher {
 
     /**
      * Requests a redraw of the scene.
+     *
+     * @param forceRedraw If `true`, draws scene immediately, without waiting for the next
+     * animation frame
      */
-    update() {
+    update(forceRedraw?: true) {
+        if (forceRedraw === true) {
+            this.redraw();
```

Comment:
in `drawFrame` (on MapView), there is some code to cancel an active requestAnimationFrame, is this necessary here also?